page.codeberg.libre_menu_editor.LibreMenuEditor.yml

```diff
@@ -0,0 +1,42 @@
+app-id: page.codeberg.libre_menu_editor.LibreMenuEditor
+runtime: org.gnome.Platform
+runtime-version: "44"
+sdk: org.gnome.Sdk
+command: libre-menu-editor
+finish-args:
+  - --env=FLATPAK_APP_DIR=/app/libre-menu-editor
```

Comment:
This can be overwritten. I recommend adding a shell wrapper that will export this variable unconditionally.

Reply:
Thank you, I will take it into consideration. Since you said "recommend", I assume that this is not a requirement. Please correct me if I am wrong.

Reply:
I think it is strange that it is an env var at all?

This is a path that will never change and isn't configurable.

Reply:
I'm of the opinion this should be part of what I'm suggesting here https://github.com/flathub/flathub/pull/4276#discussion_r1240963797

Reply:
I updated the manifest. It should be acceptable now